spec/govuk_design_system_formbuilder/builder/submit_spec.rb

```diff
@@ -7,10 +7,8 @@
     let(:args) { [method] }
     subject { builder.send(method, text) }
 
-    specify 'output should be a form group containing a submit input' do
-      expect(subject).to have_tag('div', with: { class: 'govuk-form-group' }) do |fg|
-        expect(fg).to have_tag('input', with: { type: 'submit' })
-      end
+    specify 'output should be a submit input' do
+      expect(fg).to have_tag('input', with: { type: 'submit' })
```

Comment:
Think this'll fail now `fg` isn't declared by the removed expectation block:

```suggestion
      expect(subject).to have_tag('input', with: { type: 'submit' })
```

Reply:
Oops — my mistake. Should be better now?

Reply:
:+1: